tests/parm/control_iovr.nml.IN

```diff
@@ -0,0 +1,335 @@
+&atmos_model_nml
```

Comment:
Could the existing control.nml.IN be used instead of adding this file, by making IOVR a variable in the control.nml.IN?

Reply:
Yes please. I did the same with icloud recently for control and rap for Thompson's progcld scheme.

Reply:
Absolutely. I'll take care of it.

Reply:
@DeniseWorthen @climbfuji I just took a look at this, and since default_var.sh has iovr=1 and control.nml.IN has iovr=3 hard-coded, in order to change iovr to a variable in control.nml.IN, this means that I'll need to edit every test that uses control.nml.IN to export iovr=3, right? I don't mind doing this, but I wanted to double-check because there are many tests that use control.nml.IN to be edited.

Reply:
Is iovr=1 used at all then? If not, can it be changed to 3 in default_vars?

Reply:
Operational GFS is using iovr=3. I believe other operational models are using 3 as well.  It is a good idea to set the default to 3.

Reply:
I'm assuming that it is used for all tests that don't have iovr hard-coded in their namelist and it is not set in the test file. It would be a lot of files to check, but just as an example, the control_csawmg uses the default iovr=1, so I'm guessing that there are many more.

Reply:
Setting iovr=3 in default_vars.sh is fine if that is what makes the most sense, but in order to keep other tests the same, one would then need to add iovr=1 in a large number of files.

Reply:
Just checked current operational model settings.  The "older" models lice CFS, NAM, GEFS and WDAS are still using iovr=1.  But GFS.v16 and other models planned for future implementations is or will be using iovr=3 and more advanced overlaps.

Reply:
I do not think suggestion was to change any default values, in the model nor in the default_vars.sh but to use (overwrite) the IOVR value for this test to avoid having yet another input nml template just for this test. Am I missing something?

Reply:
This is what Grant wrote earlier:

> @DeniseWorthen @climbfuji I just took a look at this, and since default_var.sh has iovr=1 and control.nml.IN has iovr=3 hard-coded, in order to change iovr to a variable in control.nml.IN, this means that I'll need to edit every test that uses control.nml.IN to export iovr=3, right? I don't mind doing this, but I wanted to double-check because there are many tests that use control.nml.IN to be edited.

Reply:
@climbfuji thanks. I see, the IOVR is not 'template' variable at all at this moment. Is the value of IOVR the only difference between contol.nml.IN and this new control_iovr.nml.IN? If it is then I think iovr should be changed to be template variable, not necessarily in the PR but eventually.

Reply:
OK, since iovr=1 and iovr=3 are both used a lot, I agree that changing default_vars.sh would be too disruptive for this PR. I'll change the tests that use control.nml.IN to export iovr=3. That is the least disruptive, IMO. The discussion to potentially change the default value of iovr can be had later if necessary.

Reply:
@DusanJovic-NOAA Yes, the only change between control.nml.IN and control_iovr.nml.IN was the iovr setting. I am just trying to add tests to exercise iovr=4 and iovr=5 to make sure they don't break in the future at the request of Mike Iacono. In fact, if adding 2 new tests is too burdensome, he says that just keeping one that excercises iovr=5 would probably be OK for now.

Reply:
I don't know what test is using iovr=1 (the current setting in default_vars). 

Isn't it either ``iovr=3`` in all namelists, except the 3 that are used by tests that set a variable ``IOVR``?

Reply:
> @DusanJovic-NOAA Yes, the only change between control.nml.IN and control_iovr.nml.IN was the iovr setting. I am just trying to add tests to exercise iovr=4 and iovr=5 to make sure they don't break in the future at the request of Mike Iacono. In fact, if adding 2 new tests is too burdensome, he says that just keeping one that excercises iovr=5 would probably be OK for now.

Since these are C96 test, do not depend on each other or other test (like restarts), and do not require new COMPILE step, I think, at this moment, we can afford adding them both.

Reply:
@DeniseWorthen Looking in tests/parm, there are a lot of namelists that don't set it (roughly half). If they're not setting iovr as an environment variable in the test file (very few do), they are using iovr=1 by default, right?

Reply:
Yes, if iovr is not specified in the namelist, it is using default value 1.

Reply:
So the model default is ``iovr=1``, correct? So any test that does not explicitly set ``iovr`` in the namelist is using the model default of ``iovr=1``. 

Only a test which has an ``iovr`` in the namelist potentially changes it from the model default value. But the only value we use in that case is ``iovr=3``. 

But I'm fine w/ the added namelist for now.